trg-checks-dashboard/web/templates/index.html.tmpl

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Don't we need a copyright header for this one?

Reply:
thanks for spotting. Added!